atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -175,15 +178,15 @@ public int execute(final AtlasDbServices services) {
                         ? sweepRunner.dryRun(tableToSweep, batchConfig, startRow.get())
                         : sweepRunner.run(tableToSweep, batchConfig, startRow.get());
                 printer.info(
-                        "Swept from {} to {} in table {} in {} ms, examined {} cell values,"
-                                + " {}deleted {} stale versions of those cells.",
-                        encodeStartRow(startRow),
-                        encodeEndRow(results.getNextStartRow()),
-                        tableToSweep,
-                        watch.elapsed(TimeUnit.MILLISECONDS),
-                        results.getCellTsPairsExamined(),
-                        dryRun ? "would have " : "",
-                        results.getStaleValuesDeleted());
+                        "{} Swept from {} to {} in table {} in {} ms, examined {} cell values,"
+                                + " deleted {} stale versions of those cells.",
+                        SafeArg.of("isDryRun", dryRun ? "[DRY RUN]" : ""),
```

Comment:
somehow this feels like it should be `true` or `false` (probably because `isX` is usually used for booleans), but I understand that this makes the substituted logs more readable. Given that we use MessageFormatter.arrayFormat(...) to print stuff to standard out, I think this is the right call!